core/client/src/main/java/alluxio/hadoop/HadoopUtils.java

```diff
@@ -200,6 +200,11 @@ public static void addSwiftCredentials(Configuration configuration) {
         && configuration.get(passwordKeyConf) == null) {
       configuration.set(passwordKeyConf, System.getProperty(passwordKeyConf));
     }
+    String swiftSimulationConf = Constants.SWIFT_SIMULATION;
+    if (System.getProperty(swiftSimulationConf) != null
```

Comment:
Not really related to this PR, but I think it may make sense to have a convenience method for this.